checker/src/org/checkerframework/checker/i18nformatter/I18nFormatterTreeUtil.java

```diff
@@ -274,7 +274,8 @@ private void initialCheck(List<? extends ExpressionTree> theargs, ExecutableElem
                                 .parse(formatforArg, flowExprContext, atypeFactory.getPath(tree));
                             paramIndex = flowExprContext.arguments.indexOf(paramArg);
                         } catch (FlowExpressionParseException e) {
-                            // errors are reported at declaration site
+                            // report errors here
+                            checker.report(e.getResult(), tree);
```

Comment:
Shouldn't this be an i18nformat.invalid.formatfor error instead of flowexpr.parse.error?